src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.scss

```diff
@@ -0,0 +1,31 @@
+@use './src/styles/vendor/variables' as vv;
+
+.danger-icon {
+  color: vv.$danger;
+}
+
+.warning-icon {
+  color: vv.$warning;
+}
+
+.border-margin {
+  height: calc(100% + 10px);
+  margin-bottom: -5px;
+  margin-left: -5px;
+  margin-top: -5px;
+}
+
+.danger-border {
+  border-left: vv.$danger 1px solid;
+}
+
+.warning-border {
+  border-left: vv.$warning 1px solid;
+}
```

Comment:
I like the style: I'd make it slightly thicker. Have you checked [this resource](https://www.patternfly.org/v4/guidelines/colors/#status-and-state-colors)? And why not moving to a top-level styling SCSS?

Reply:
I'll leave you some pictures with 2px, 3px and 4px(default in top-level)
![image](https://user-images.githubusercontent.com/30913090/112277550-2a679280-8c82-11eb-9ba4-388cc3ff6ec6.png)
![image](https://user-images.githubusercontent.com/30913090/112277623-394e4500-8c82-11eb-8543-5a1a2747bea9.png)
![image](https://user-images.githubusercontent.com/30913090/112277684-48cd8e00-8c82-11eb-97a5-c8e997f0f13f.png)
Which one do you like? In my case 2px is what I find to suit better.

Reply:
Based on [this](https://www.patternfly.org/v4/components/table) I see Patternfly recommends 3px with. I think 2px is too subtle and 4px too harsh :P What others think?: @nizamial09, as our downstream branding expert (you already added this to downstream, right)?

Additionally, in order to keep the left-alignment, you need to add that border-width to all rows, but only color the border in the highlighted rows.

Reply:
I've added this in some parts. I like the 3px one here.I've mostly added this in notifiation sidebar and it was 4px (3px is too subtle over there) @pereman if you are going with 3px please align the Usernames without the left border with the ones that has left border.